service/src/java/org/apache/hive/service/server/HiveServer2.java

```diff
@@ -217,9 +217,9 @@ public void run() {
       }
     };
     if (isHTTPTransportMode(hiveConf)) {
-      thriftCLIService = new ThriftHttpCLIService(cliService, oomHook);
+      thriftCLIService = new ThriftHttpCLIService(cliService);
```

Comment:
Remove the definition of oomHook as it is not used anywhere else.

Reply:
Done this